css/style.css

```diff
@@ -0,0 +1,464 @@
+  width: 51px;
+  height: 43px;
+  margin: 0 0 0 12px;
+  padding: 0;
+}
+
+.text {
+  font-size: 20px;
+  line-height: 24px;
+  font-style: normal;
+  font-weight: 600;
+  text-align: center;
+}
+
+.time-speed__text {
+  display: flex;
+  align-items: flex-end;
+  width: 37px;
+  height: 41px;
+  margin: 0 0 0 6px;
+  padding: 0;
+}
+
+.subtext {
+  font-size: 12px;
+  line-height: 15px;
+  font-style: normal;
+  font-weight: normal;
+}
+
+.time-speed__subtext {
+  display: flex;
+  width: 80px;
+  height: 15px;
+  margin: -15px 0 0 60px;
+}
+
+.distance {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: 145px;
+  height: 58px;
+  margin: 0 82px 0 0;
+  padding: 0;
+}
+
+.distance__line {
+  display: flex;
+  width: 31px;
+  height: 58px;
+  margin: 0 8px 0 0;
+  background: url(https://i.ibb.co/DCQBjjV/Road-icon.png) no-repeat;
+  background-size: 100%;
+  background-position: center;
+}
+
+.distance__numeral {
+  display: flex;
+  width: 63px;
+  height: 43px;
+  margin: 0 6.5px 0 0;
+  padding: 0;
+}
+
+.distance__text {
+  display: flex;
+  width: 29px;
+  height: 25px;
+  margin: 15px 0 6.5px 0;
+  padding: 0;
+}
+
+.distance__subtext {
+  display: flex;
+  width: 67px;
+  height: 15px;
+  font-style: normal;
+  font-weight: normal;
+  text-align: left;
+  margin: -16px 0 0 41px;
+  padding: 0;
+}
+
+.test-drive {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+/* руль тест */
+
+.test-drive__rudder {
+  display: flex;
+  width: 22px;
+  height: 58px;
+  margin: 0 6px 0 0;
+  padding: 0;
+  background: url(https://i.ibb.co/wWgHDvw/steering-wheel.png) no-repeat;
+  background-size: 100%;
+  background-position: center;
+}
+
+.test-drive__link {
+  display: flex;
+  width: 89px;
+  height: 18px;
+  font-size: 13px;
+  line-height: 16px;
+  font-style: normal;
+  font-weight: bold;
+  text-decoration: none;
+  color: #000;
+  margin: 0 40px 0 0;
+  text-transform: uppercase;
+}
+
+.test-drive__link:hover {
+  display: flex;
+  font-size: 13px;
+  line-height: 16px;
+  font-weight: bold;
+  text-align: center;
+  text-decoration: none;
+  color: #f00;
+  cursor: pointer;
+  padding: 0;
+  border: 0;
+}
+
+/* Кнопка Заказать */
+
+.button {
+  display: flex;
+  width: 160px;
+  height: 40px;
+  font-size: 13px;
+  line-height: 16px;
+  font-family: "Montserrat", sans-serif;
+  text-align: center;
+  font-weight: bold;
+  text-transform: uppercase;
+}
+
+.test-drive__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 13px;
+  text-align: center;
+  font-weight: bold;
+  border: 2px solid #f00;
+  color: #f00;
+  background-color: #fff;
+}
+
+.test-drive__button:hover {
+  font-size: 14px;
+  color: #f00;
+  border: 3px solid #f00;
+  background-color: #fff;
+  cursor: pointer;
+  font-weight: bold;
+  padding: 0;
+}
+
+.test-drive__button:focus {
+  display: flex;
+  color: #ff0000;
```

Comment:
Codacy found an issue: [Expected "#ff0000" to be "#f00" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5710464)